src/plugins/intel_gpu/tests/unit/test_cases/permute_gpu_test.cpp

```diff
@@ -855,6 +855,68 @@ TEST(permute_gpu_f32, 6D_reshape_permute_reshape)
         ASSERT_EQ(expected_out[i], output_ptr[i]);
     }
 }
+
+TEST(permute_gpu_f32, 6D_reshape_permute_reshape_trucation_mode)
```

Comment:
It will cause confusion to add this test in permute tests set. 
please add the test under test/unit/passes/handle_reshape.cpp

Reply:
done